src/test/java/be/syntra/devshop/DevshopFront/controllers/UserControllerTest.java

```diff
@@ -82,11 +93,30 @@ void canDoRemoveOneFromProductTotalInCart() throws Exception {
 
         // then
         getResult
-                .andExpect(redirectedUrl("/users/cart/overview"));
+                .andExpect(redirectedUrl("/users/cart/detail"));
 
         verify(cartService).removeOneFromProductInCart(2L);
     }
 
+    @Test
+    @WithMockUser(roles = {"USER"})
```

Comment:
@WithMockUser's default value = (roles = {"USER"}). Remove redundant code please

Reply:
fix for all tests in entire class please